packages/rest/src/controllers/connections/ConnectionController.ts

```diff
@@ -94,6 +117,27 @@ export class ConnectionController {
     }
   }
 
+  /**
+   * Receive connection invitation as invitee by invitationUrl and create connection. If auto accepting is enabled
+   * via either the config passed in the function or the global agent config, a connection
+   * request message will be send.
+   */
+  @Post('/receive-invitation-url')
+  public async receiveInvitationByUrl(@Body() invitationByUrlRequest: ReceiveInvitationByUrlRequest) {
+    const { invitationUrl, ...config } = invitationByUrlRequest
+
+    try {
+      const base64 = invitationUrl.split('=')[1]
+      const invitation = JsonEncoder.fromBase64(base64)
+      const inv = JsonTransformer.fromJSON(invitation, ConnectionInvitationMessage)
+      const connection = await this.agent.connections.receiveInvitation(inv, config)
```

Comment:
we have a `receiveInvitationFromUrl` in AFJ, can we use that method? (it already includes the parsing logic for extracting the `?c_i=` parameter)